apps/computeVSD/computeVSD.cpp

```diff
@@ -195,10 +199,37 @@ class ComputeVSD : public CommandLineApplication
 
         VolumePtr output = source->GetOutput();
         output->SetRegions( fivox::FloatVolume::RegionType( vIndex, vSize ));
-        output->SetSpacing( volumeHandler.computeSpacing( ));
 
-        const fivox::AABBf& bbox = vsdLoader->getBoundingBoxSomas();
-        output->SetOrigin( volumeHandler.computeOrigin( bbox.getCenter( )));
+        fivox::AABBf bboxSomas;
+        const auto& somaPositions = vsdLoader->getSomaPositions();
+        for( const auto& position : somaPositions )
+            bboxSomas.merge( position );
+
+        const auto spacing = volumeHandler.computeSpacing();
+        const auto origin = volumeHandler.computeOrigin( bboxSomas.getCenter());
+
+        if( _vm.count( "soma-pixels" ))
+        {
+            const auto& fileName = _vm["soma-pixels"].as< std::string >();
+            std::ofstream file( fileName );
+
+            size_t i = 0;
+            const auto& gids = vsdLoader->getGIDs();
+            for( const auto& gid : gids )
+            {
+                const auto pos = somaPositions[i];
+                file << gid << " " << pos << ": "
```

Comment:
I guess the user wants that format and he has python magic to know the semantics. Putting a least a header information what the different fields mean and a version tag somewhere to be a little bit safe?